### ISSUE TITLE
Add other rail companies

### ISSUE THREAD

**Issue by Enrico204** [MarcoBuster/OrarioTreniBot#12]
It would be useful to have also as many as possible of these:

 - TreNord (trenord.it)
 - NTV (italotreno.it)
 - SNCF/Interrail (sncf.com)
 - EAV (eavsrl.it)
 - Ferrovie del Sud-Est (fseonline.it)
 - Ferrovie del Gargano (ferroviedelgargano.com)

(I think that NTV, TreNord and SNCF are widely used in northern regions)

**Comment by MarcoBuster** [comment]
Good idea! We may be forced to use HTML parsing, but it's not a problem!
_(TreNord is already included by viaggiatreno.it system.)_

**Comment by Enrico204** [comment]
I think that we may create a middleware in python that exposes REST/JSON API for HTML-only websites in order to keep this bot clean and simple :-)

**Comment by MarcoBuster** [comment]
Great! A master API for all rail companies. It can be a new project.
